### PR TITLE
Read ngrams from postgres db

### PR DESCRIPTION
**Pull request by jboileau99**

### CODE REVIEW

File: src/data/malamud_dataset.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import gensim
+import connectorx as cx
+
+class MalamudDataset:
+    def __init__(self, postgres_uri: str):
+        self.df = cx.read_sql(postgres_uri, "SELECT * FROM docs.doc_ngrams_0")
+        self.ngrams = self.df.ngram.apply(gensim.utils.simple_preprocess)
```
Comment (christianjans) [inline src/data/malamud_dataset.py]:
```suggestion
        self.ngrams = self.df.ngram_lc.apply(gensim.utils.simple_preprocess)
```

Comment (christianjans) [inline src/data/malamud_dataset.py]: I wonder if we will have a problem storing all of these in memory eventually. Any thoughts on this or ways to circumvent it?

File: README.md [hunk 2]
```diff
@@ -2,14 +2,26 @@
...
 ```
 
-# Helpful Notes for Later
+## Database setup
+
+The model training script expects ngram data to be stored in `docs.doc_ngram_0` (`docs` schema, `doc_ngram_0` table) within a postgres database. For the script to connect to this database, add a .env file in the `/src` directory that sets the `PG_URI` variable the [connection string](https://stackoverflow.com/questions/3582552/what-is-the-format-for-the-postgresql-connection-string-url) for the database.
+
+An example .env file could be:
+
+```
```
Comment (christianjans) [inline README.md:19]: Just being pedantic

```suggestion
```sh
```

File: src/data/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+On the ISAIC vm, there is postgres DB with the first 10 million rows of the first chunk of ngrams.
+
+DB Name: `malamud`
+Table: `docs.doc_ngrams_0`
+
+## Creating DB from file
+
+Command to load the first ~10mil lines from the ngram dump into the `malamud` db:
+
+```
```
Comment (christianjans) [inline src/data/README.md:12]: Just being pedantic to get text colouring

```suggestion
```sh
```

**Review by christianjans — COMMENTED** [review]
Looks incredible! Just a couple comments.

File: src/data/malamud_dataset.py (hunk 1 above)
Comment (christianjans) [reply src/data/malamud_dataset.py]: Ahh, resolved now! Thanks!

Comment (jboileau99) [reply src/data/malamud_dataset.py]: @christianjans Should be fixed now! I added a `chunk_size` param so we can experiment with different numbers of rows to see what's fastest. Currently it's training with chunk_size = 1M and the memory holds steady

**Pull request merged**